### PR TITLE
splits conventions off, adds router ref

### PR DESCRIPTION
**Pull request by andrewpmartinez**

### CODE REVIEW

File: docusaurus/docs-ops/30-configuration/conventions.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+- `transport_udp` - listener, dialer - allows ingress UDP connections to request a circuit for a service and for traffic
+  to dial to an underlay UDP service at a `<host>:<port>` address
+- `edge` - listener, dialer - allows multiplexed ingress connections from SDKs and connections to other SDKs hosting
+  services
+
+### xgress Options
+
+Each xgress component can have its own options in addition to the following shared options:
+
+- `mtu` - (optional, 640*1024) unused
```
Comment (plorenz) [inline docusaurus/docs-ops/30-configuration/conventions.md:218]: Maybe make a note that randomDrops and drop1InN are used for testing?

File: docusaurus/docs-ops/30-configuration/router.md [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+other address. The `advertise` value should be externally routable.
+
+- `binding` - (required) the name of a [xgress component](conventions.md#xgress-components) that will provide the server
+  side logic for the listener
+- `bind` - (required) the [address](conventions.md#addressing) that the listener should listen on
+- `advertise` (required) the `<host>:<port>` combination that external resources should use to reach this listener
+- `costTags` - (optional) an array of tags used during link cost calculations
+- `options` - (optional) options specified by the component specifically in addition to
+  the [shared options](conventions.md#xgress-options)
+
```
Comment (plorenz) [inline docusaurus/docs-ops/30-configuration/router.md:303]: empty section

**Review by plorenz — APPROVED** [review]

File: docusaurus/docs-ops/30-configuration/conventions.md (hunk 1 above)
Comment (andrewpmartinez) [reply docusaurus/docs-ops/30-configuration/conventions.md:218]: Added.

File: docusaurus/docs-ops/30-configuration/router.md (hunk 2 above)
Comment (andrewpmartinez) [reply docusaurus/docs-ops/30-configuration/router.md:303]: Noted, fleshing it out now.

**Pull request merged**